chart-diagrams/Chart-diagrams.cabal

```diff
@@ -32,7 +32,7 @@ library
                , diagrams-lib >= 1.2 && < 1.5
                , diagrams-svg >= 1.4 && < 1.5
                , diagrams-postscript >= 0.7 && < 1.5
-               , SVGFonts >= 1.4 && < 1.7
```

Comment:
Doesn't the lower version bound for this need to be 1.7, as for versions <1.7 F.lin and F.bit are pure?

Reply:
Yes, will change.